docs/communication_protocols/hobby_protocol.rst

```diff
@@ -10,6 +10,11 @@ Hobby Protocols
 Hobby protocols refers to an array of different protocols commonly used by drone hobbyists for controlling ESCs. Vertiq modules support a selection of the 
 most popular of these protocols. Some details on these protocols and how Vertiq modules support them is provided below.
 
+.. note::
+
+    Please note that no two hobby protocols can be used together as :ref:`redundant throttle sources <redundant_throttle_manual>`. Only one 
+    hobby protocol can be accepted at any time.
```

Comment:
Maybe worth linking to where we define hobby protocols here, or at least giving an example of what they are e.g. "such as PWM and DSHOT", because people don't all necessarily know what the "hobby protocols" term means

Reply:
this is where we define what a hobby protocol is...
![image](https://github.com/user-attachments/assets/e222b3ad-1dc3-41f5-934b-954566825ebf)

Reply:
Ok, missed what file it was in